content/en/user-guide/aws/lambda/index.md

```diff
@@ -123,38 +162,136 @@ $ aws lambda add-layer-version-permission
+- [Lambda Code Mounting and Debugging](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-mounting-and-debugging) demonstrates how to debug Lambda functions locally using code mounting.
+- [Lambda Function URL](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-function-urls) shows how to use HTTP to invoke a Lambda function via its Function URL.
+- [Lambda Layers](https://github.com/localstack/localstack-pro-samples/blob/master/serverless-lambda-layers) demonstrates how to use Lambda layers, which are reusable packages of code that can be shared across multiple functions.
+- [Lambda PHP/Bref](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-php-bref-cdk-app) shows how to use PHP/Bref with and without fpm, using the Serverless framework and AWS CDK.
+- [Lambda Container Images](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-container-image) demonstrates how to use Lambda functions packaged as container images, which can be built using Docker and pushed to a local ECR registry.
+- [Lambda XRay Tracing](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-xray) shows how to enable AWS X-Ray tracing for Lambda functions using the Serverless framework.
+
+## Troubleshooting
+
+### Docker not available
+
+In the old Lambda provider, Lambda functions were executed within the LocalStack container using the local executor mode. This mode was used as a fallback if the Docker socket was unavailable in the LocalStack container. However, many users inadvertently used the local executor mode instead of the intended Docker executor mode, which caused unexpected behavior.
+
+If you encounter the following error message, you may be using the local executor mode:
+
+{{< tabpane >}}
+{{< tab header="LocalStack Logs" lang="shell" >}}
+Lambda 'arn:aws:lambda:us-east-1:000000000000:function:my-function:$LATEST' changed to failed. Reason: Docker not available
+...
+raise DockerNotAvailable("Docker not available")
+{{< /tab >}}
+{{< tab header="AWS CLI" lang="shell" >}}
+An error occurred (ResourceConflictException) when calling the Invoke operation (reached max retries: 0): The operation cannot be performed at this time. The function is currently in the following state: Failed
+{{< /tab >}}
+{{< tab header="SAM" lang="shell" >}}
+Error: Failed to create/update the stack: sam-app, Waiter StackCreateComplete failed: Waiter encountered a terminal failure state: For expression "Stacks[].StackStatus" we matched expected path: "CREATE_FAILED" at least once
+{{< /tab >}}
+{{< /tabpane >}}
+
+To fix this issue, add the Docker volume mount `/var/run/docker.sock:/var/run/docker.sock` to your LocalStack startup. Refer to our [sample `docker-compose.yml` file](https://github.com/localstack/localstack/blob/master/docker-compose.yml) as an example.
+
+### Lambda functions not found
```

Comment:
These headers need to match the original migration docs (here `### Function in Pending state`), otherwise it breaks the pro-active link in the LocalStack logs: https://github.com/search?q=repo%3Alocalstack%2Flocalstack+https%3A%2F%2Fdocs.localstack.cloud%2Freferences%2Flambda-provider-v2&type=code